examples/asr/experimental/audio_processing/audio_to_audio_enc_mask_dec.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Training the model
+
+Basic run (on CPU for 50 epochs):
+    python examples/asr/experimental/audio_processing/audio_to_audio_enc_mask_dec.py \
+        # (Optional: --config-path=<path to dir of configs> --config-name=<name of config without .yaml>) \
+        model.train_ds.manifest_filepath="<path to manifest file>" \
+        model.validation_ds.manifest_filepath="<path to manifest file>" \
+        trainer.devices=1 \
+        trainer.accelerator='cpu' \
+        trainer.max_epochs=50
+
+PyTorch Lightning Trainer arguments and args of the model and the optimizer can be added or overriden from CLI
+"""
+import pytorch_lightning as pl
```

Comment:
can you name this script more specific than "audio_to_audio_enc_mask_dec.py". usually the scripts in examples folder are per task. if this is for enhancement, may be just name, speech_enhancement.py